test/integration/suites/spire-server-cli/04-bootstrap-agents

```diff
@@ -5,9 +5,9 @@ docker-compose exec -T spire-server \
     /opt/spire/bin/spire-server bundle show > conf/agent/bootstrap.crt
 
 # Set conf files for each agent
-cp -R conf/agent conf/agent-1
-cp -R conf/agent conf/agent-2
-cp -R conf/agent conf/agent-3
+cp -R conf/agent/ conf/agent-1
```

Comment:
is `/` required?

Reply:
yes it appears to be, at least on Big Sur release. Didn't work without it